config.py

```diff
@@ -1,7 +1,14 @@
 import logging
 import os
 
-COBOT_ROOT = os.environ.get('COBOT_ROOT', os.getcwd())
+# Deprecate COBOT_ROOT
+BOT_ROOT = os.environ.get('BOT_ROOT', os.getcwd())
+if 'BOT_ROOT' not in os.environ:
+    os.environ.setdefault('BOT_ROOT', 'COBOT_ROOT')
+    logging.warning()
+    logging.warning('Deprecation warning: environment variable {} is'
+                    + 'replaced by {}.'
+                    .format('COBOT_ROOT', 'BOT_ROOT'))
```

Comment:
Why are you using ``+`` in between the string? You can simply write ``'Deprecation warning: environment variable {} is replaced by {}.'.format(..)``. This would be better.

Reply:
And is there any need of using ``logging.warning()``? There's no need of empty warning I suppose as you are supplying the warning on line ``9``.